core/bones/base.py

```diff
@@ -74,7 +97,7 @@ def __init__(
         languages: Union[None, List[str]] = None,
         multiple: Union[bool, MultipleConstraints] = False,
         params: Dict = None,
-        readOnly: bool = False,
+        readOnly: bool = None,  # fixme: Rename into readonly (all lowercase!) soon.
```

Comment:
This is set to `None` to distinguished an explicitly specified or unspecified value, see below.